src/Knp/Rad/ResourceResolver/ParameterCaster/ApplicationParameterCaster.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Knp\Rad\ResourceResolver\ParameterCaster;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Doctrine\Instantiator\Exception\InvalidArgumentException;
+
+class ApplicationParameterCaster
+{
+    public function __construct(ContainerBuilder $container)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * @return bool
+     */
+    public function supports($string)
+    {
+        return 1 === preg_match('/^%(\w|\.)+%$/', $string);
+    }
+
+    /**
+     * @return mixed
+     */
+    public function cast($string)
+    {
+        try {
+            return $this->container->getParameter($string);
+        } catch (InvalidArgumentException $exception) {
```

Comment:
@umpirsky Works with get method, but not getParameter one.

'''
    /**
     * Gets a parameter.
     *
     * @param string $name The parameter name
     *
     * @return mixed The parameter value
     *
     * @throws InvalidArgumentException if the parameter is not defined
     *
     * @api
     */
    public function getParameter($name);
'''